goosebit/models.py

```diff
@@ -136,5 +138,12 @@ def path(self):
         return Path(url2pathname(unquote(urlparse(self.uri).path)))
 
     @property
-    def local(self):
+    def local(self) -> bool:
         return urlparse(self.uri).scheme == "file"
+
+    @property
+    def path_user(self) -> str:
+        if self.local:
+            return self.path.name
+        else:
+            return self.uri
```

Comment:
This might be a bit long, maybe want to use the end of the URI path or something?

Reply:
That we had before (just the "filename"). For remote firmware, this filename could always be the same - as the difference is in the directory structure. Also helpful to have one place where the full url can be seen IMHO.